src/resources/views/crud/fields/checklist.blade.php

```diff
@@ -25,11 +25,19 @@
 
 @include('crud::fields.inc.wrapper_start')
     <label>{!! $field['label'] !!}</label>
+    <div class="row">
+        <div class="col-sm-4">
+            <label class="font-weight-normal">
+                <input type="checkbox"  id="select-all" {{ count($field['options']) == count($field['value']) ? 'checked' : '' }}> 
+                <strong> Select all </strong>
+            </label>
+        </div>
+    </div>
     @include('crud::fields.inc.translatable_icon')
 
     <input type="hidden" value='@json($field['value'])' name="{{ $field['name'] }}">
 
-    <div class="row">
+    <div class="row" id="checkbox-wrapper">
```

Comment:
We can't use `id` inside fields, only `class` or `data-attributes`. We need to account for the fact that there might be two `checklist` fields in a form. If that happens, and we use IDs, clicking one "select all" might select everything in all `checklist` fields, which is not the intended behaviour.

Reply:
In this particular case, I think a `checkbox-wrapper` class would do better.